pymc/model.py

```diff
@@ -548,6 +548,7 @@ def __init__(
         self,
         name="",
         coords=None,
+        coords_mutable=None,
         check_bounds=True,
         *,
```

Comment:
I'll apply this change before triggering the tests. The reason being that only by placing the new parameter behind the `*,` we can avoid a change of signature.

```suggestion
        check_bounds=True,
        *,
        coords_mutable=None,
```